packages/vue/src/components/organisms/SfHero/_internal/SfHeroItem.vue

```diff
@@ -10,22 +10,28 @@
         <h1 v-if="title" class="sf-hero-item__title">{{ title }}</h1>
       </slot>
       <!--@slot Call to action section. Slot content will replace default SfButton component-->
-      <slot name="call-to-action" v-bind="{ buttonText }">
+      <slot name="call-to-action" v-bind="{ buttonText, link }">
         <div v-if="buttonText" class="sf-hero-item__button">
-          <SfButton>
+          <SfButton :link="link">
```

Comment:
I think it would be good to have a condition here to avoid button in button situaction  :)